### PR TITLE
Initial instructions on filing HST quaterly

### PR DESCRIPTION
**Pull request by YurkoWasHere**
Instructions on how to prepare, file and pay HST quarterly

### CODE REVIEW

File: guides.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ our values.
 - [Inactivating Membership](#inactivating-membership)
 - [Meetings](#meetings)
 - [Payroll](#payroll)
+- [Reporting HST]
```
Comment (YurkoWasHere) [inline guides.md:29]: Not sure how this works:

```suggestion
- [Reporting HST](#reporting-hst)
```

Comment (benhylau) [inline guides.md:29]:
```suggestion
- [Reporting HST](#reporting-hst)
```

File: guides.md [hunk 2]
```diff
@@ -614,7 +615,69 @@ If this is the first time you use Wagepoint, please first review the [Running yo
...
+5. Enter the `Access Code` from Passbolt's password field
+6. Enter the first day and last day of the quarter you are reporting for
+7. Confirm no checkboxes need to be checked (as of September 2021, no checkboxes are required to be checked)
+8. Press `Next`
+9. Enter line `101`, `105` and `108` from the bookkeeper's email. Make sure you do not have any leading or trailing spaces and only one decimal place
+10. Click `Next`
+11. Review summary, and check line `109` and `114` or `115` for matching values from the bookkeer's email
+12. Check `Yes` regarding payment (if applicable)
+13. Check the box under `Certification`
+14. Click `File`
```
Comment (YurkoWasHere) [inline guides.md:657]: Need to confirm the text on this button

File: guides.md [hunk 3]
```diff
@@ -614,7 +615,69 @@ If this is the first time you use Wagepoint, please first review the [Running yo
...
+15. Click `Save as PDF` at the bottom of the page
+16. Rename file in correct format with today's date and last day of quarter `yyyy`-`mm`-`dd`-hst-ending-20`xx`-`xx`-`xx`-netfile.pdf`
+17. Uplod file to Googe Drive until `Finance/HST`
+
+### Payment of HST
+
+If payment is required for HST, you will need to coordinate with another member that has signing authority at the bank. The payment must be confirmed by the 2nd person within 24 hours.
```
Comment (YurkoWasHere) [inline guides.md:664]: I think it may be 48  hours not 24?

File: guides.md [hunk 4]
```diff
@@ -614,7 +615,69 @@ If this is the first time you use Wagepoint, please first review the [Running yo
...
+2. Press the `Ready to file` button
+3. Press the `Continue` button at the bottom of the page
+4. Enter the `Business Number` from Passbolt. Note the number does not include spaces. Note that on the site the number is broken into two fields seperated at `RT`
+5. Enter the `Access Code` from Passbolt's password field
+6. Enter the first day and last day of the quarter you are reporting for
+7. Confirm no checkboxes need to be checked (as of September 2021, no checkboxes are required to be checked)
+8. Press `Next`
+9. Enter line `101`, `105` and `108` from the bookkeeper's email. Make sure you do not have any leading or trailing spaces and only one decimal place
+10. Click `Next`
+11. Review summary, and check line `109` and `114` or `115` for matching values from the bookkeer's email
```
Comment (benhylau) [inline guides.md:654]:
```suggestion
11. Review summary, and check line `109` and `114` or `115` for matching values from the bookkeeper's email
```

File: guides.md [hunk 5]
```diff
@@ -614,7 +615,69 @@ If this is the first time you use Wagepoint, please first review the [Running yo
...
+12. Check `Yes` regarding payment (if applicable)
+13. Check the box under `Certification`
+14. Click `File`
+15. Click `Save as PDF` at the bottom of the page
+16. Rename file in correct format with today's date and last day of quarter `yyyy`-`mm`-`dd`-hst-ending-20`xx`-`xx`-`xx`-netfile.pdf`
+17. Uplod file to Googe Drive until `Finance/HST`
```
Comment (benhylau) [inline guides.md:660]:
```suggestion
17. Upload file to Google Drive until `Finance/HST`
```

File: guides.md [hunk 6]
```diff
@@ -614,7 +615,69 @@ If this is the first time you use Wagepoint, please first review the [Running yo
...
+
+1. Login to Desjardins
+2. Select the `Payments` button on the right, a pulldown menu will appear
+3. Select `Government remittances - Can-Act`
+4. Select `Confirm` on the popup
+5. Select `Federal - GST/HST`
+6. Select `Pay` at the top bar
+7. Enter last day of the quarter as `Period Ending`
+8. Enter amount from Line 115 as `Amount owing`
+9. Select `next`
```
Comment (benhylau) [inline guides.md:674]:
```suggestion
9. Select `Next`
```

File: guides.md [hunk 7]
```diff
@@ -614,7 +615,69 @@ If this is the first time you use Wagepoint, please first review the [Running yo
...
+
+### TODO
+- [ ] Verify and Submit instrucitions
+- [ ] Confirmation
+- [ ] 2nd Person accept payment
+- [ ] Step 14 of filling - confirm name of button
```
Comment (benhylau) [inline guides.md:680]: Get rid of this before merge.

**Review by benhylau — CHANGES_REQUESTED** [review]
Can merge after my comments.

File: guides.md [hunk 8]
```diff
@@ -614,6 +615,68 @@ If this is the first time you use Wagepoint, please first review the [Running yo
...
+12. Check `Yes` regarding payment (if applicable)
+13. Check the box under `Certification`
+14. Click `Submit`
+15. Click `Save as PDF` at the bottom of the page
+16. Rename file in correct format with today's date and last day of quarter `yyyy`-`mm`-`dd`-hst-ending-20`xx`-`xx`-`xx`-netfile.pdf`
+17. Upload file to Google Drive until `-Backoffice/Finance/HST`
+
+### Payment of HST
+
+If payment is required for HST, you will need to coordinate with another member that has signing authority at the bank. The payment must be confirmed by the 2nd person within 24 hours.
```
Comment (YurkoWasHere) [inline guides.md:664]:
```suggestion
If payment is required for HST, you will need to coordinate with another member that has signing authority at the bank. The payment must be confirmed by the 2nd person within 48 hours.
```

Comment (YurkoWasHere) [reply guides.md:664]: Good to merge just a small change 24->48 hours

**Comment by YurkoWasHere** [comment]
Gdod tho merge

File: guides.md [hunk 9]
```diff
@@ -26,6 +26,7 @@ our values.
 - [Inactivating Membership](#inactivating-membership)
 - [Meetings](#meetings)
 - [Payroll](#payroll)
+- [Reporting HST](#reporting-hst)
 - [Initatives](#initatives)
```
Comment (benhylau) [inline guides.md:30]:
```suggestion
- [Initiatives](#initiatives)
```

File: guides.md [hunk 10]
```diff
@@ -614,6 +615,68 @@ If this is the first time you use Wagepoint, please first review the [Running yo
...
+8. Enter amount from Line 115 as `Amount owing`
+9. Confirm right amount displays in `Total payment` field
+10. Select `Next`
+11. Confirm values are correct
+12. Select `Submit`
+13. Confirm message `The transaction is now in Pending Approval status and requires 1 approver(s).` Displays at the top
+14. Click `Done`
 
 
 ## Initatives
```
Comment (benhylau) [inline guides.md:682]:
```suggestion
## Initiatives
```

**Review by benhylau — APPROVED** [review]

**Pull request merged**